.github/workflows/pages-check-links.yml

```diff
@@ -0,0 +1,39 @@
+# This is a basic workflow to help you get started with Actions
+
+name: pages-check-links
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1.113.3
+        with:
+          ruby-version: 2.7.x
+      
+      - name: Setup Rubygems, Bundler, jekyll
+        run: | 
+          gem update --system --no-document
+          gem update bundler --no-document
+          gem install jekyll bundler
+          bundle install
+      
+      - name: Build jekyll website with drafts
+        run: cd docs/ && bundle exec jekyll build --drafts
+        
+      - name: Check for broken links
+        run: |
+          bundle exec htmlproofer --log-level :debug ./_site --swap-urls '^/thundernetes/:/' --ignore-status-code 403
```

Comment:
why is 403 ignored?

Reply:
Some external links throw 403 but then redirect, like this one for example https://mvnrepository.com/artifact/com.playfab/gameserverSDK